src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,11 @@
+import { LoginForm } from './LoginForm';
```

Comment:
Please put the component in the folder 😊

```suggestion
import { LoginForm } from './components/LoginForm';
```

Reply:
fixed